zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/tenant/TenantController.java

```diff
@@ -78,4 +79,56 @@ private CompletableFuture<ResponseEntity<Object>> updateTenant(final TenantDTO t
                 .updateTenant(tenantDTO),
         ResponseMapper::toTenantUpdateResponse);
   }
+
+  // The API will accept only a single userKey for now to maintain atomicity and align
+  // with REST principles.
+  // Bulk operations would require adapting the broker request and adding a new event like
+  // ENTITIES_ADDED, which is out of scope for this iteration.
```

Comment:
I'm not sure this should be a comment. Maybe we can move it into a ticket? If you want to keep the comment, let's remove the reference to the current iteration (which will not age well).